### DISCUSSION TITLE
Add Tauri as an alternative to Electron

### DISCUSSION THREAD

**Discussion by lodembeep** [quasarframework/quasar#17696] [Ideas / Proposals]
Tauri 2.0 | Tauri

https://v2.tauri.app/

**Documentation** : https://v2.tauri.app/start/

Hello everyone, I would like to know if it is possible to add support for Tauri since it offers a lot of advantages over electron. I am not affiliated to Tauri in any way but over the last years I used Electron a lot, and Tauri seems to be a great alternative that will surely shine in the next few years/months.

It could be another option under the vite setup (preferably under webpack too).

PS: Seems like nobody asked about Tauri since 2022, now they have v2 so it may need some reassessment.

**Comment by ELABDELLAOUI-ismail** [comment]
yees, i think adding support for Tauri would be great! it s lightweight architecture, small bundle size, and better performance make it a strong alternative as i see

**Comment by MohammedAmine-ELHARCHI** [comment]
Completely agree with this suggestion. Tauri has made significant strides with the release of version 2.0, and its lightweight nature and security focus make it a compelling alternative to Electron.

**Comment by yacinova** [comment]
Totally agree! Tauri is lightweight and offers impressive performance compared to Electron—having it as an option under the Vite setup would be a game-changer. Hope this gets some traction!

**Comment by zanzara** [comment]
I agree with u all, but it will be indispensable to learn Rust then.
Real-World Desktop appclications can have pretty much specific code in a  `electron-main.js` , which the Tauri-API doesn't cover.

**Reply by Creatozz** [reply]
Great suggestion. Many now move to tauri who never used quasar. Could be a winning combination, especially for those who prefer vue.
